.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Codecov
+on:
+  push:
+    branches: [ git_workflow ]
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Install dependencies
+        run: npm install
+      - name: Run tests
+        run: npm test 
+      - name: Run coverage
+        run: npm run coverage 
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+           token: ${{ secrets.CODECOV_TOKEN }}
+
+  publish-npm:
+    needs: codecov
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
```

Comment:
Just a question, we need separate versions here?

Reply:
updated the above node version `16`

Reply:
convert this to array to test for 12 and 14

Reply:
added also try in 18 and 19 and it's working, should we keep 18 and 19 or remove it?

Reply:
lets keep it if not breaking anything